mllib/src/main/scala/org/apache/spark/mllib/clustering/DistanceMeasure.scala

```diff
@@ -117,6 +117,17 @@ private[spark] abstract class DistanceMeasure extends Serializable {
     packedValues
   }
 
+  def findClosest(
+      centers: Array[VectorWithNorm],
+      statistics: Option[Array[Double]],
+      point: VectorWithNorm): (Int, Double) = {
+    if (statistics.nonEmpty) {
+      findClosest(centers, statistics.get, point)
+    } else {
+      findClosest(centers, point)
+    }
+  }
```

Comment:
Shall we add the function description like the other existing `def findClosest` functions?

Reply:
good point. I will update this PR